src/main/java/org/broadinstitute/consent/http/resources/SwaggerResource.java

```diff
@@ -133,12 +133,12 @@ private String getInitializer() {
           });
           ui.initOAuth({
             clientId: "OAUTH_CLIENT_ID",
```

Comment:
the `clientId` here is going to be a problem. Out of the box, it looks like the swagger ui supports only 1 client id but we have 2. TDR and others have an [index.html](https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/resources/templates/index.html) that adds `insertClientIdsPlugin` but consent has a later version of swagger ui and I did not explore how it might work.

Reply:
We'll file a ticket to address this in a separate PR - it shouldn't be too involved to migrate to our own index like TDR does.